src/Files.App/Views/Properties/HashesPage.xaml

```diff
@@ -22,25 +21,10 @@
 	<Grid x:Name="RootGrid">
 
 		<!--  Loading  -->
-		<StackPanel
-			x:Name="LoadingStatePanel"
-			HorizontalAlignment="Center"
-			VerticalAlignment="Center"
-			x:Load="{x:Bind HashesViewModel.IsLoading, Mode=OneWay}"
-			Spacing="8">
-			<ProgressRing HorizontalAlignment="Center" IsIndeterminate="True" />
-			<TextBlock
-				x:Name="LoadingTextBlock"
-				HorizontalAlignment="Center"
-				Style="{StaticResource BodyStrongTextBlockStyle}"
-				Text="{helpers:ResourceString Name=Loading}" />
-		</StackPanel>
```

Comment:
Why removed?

Reply:
Loading the file and calclulating the hash is now processed when `isEnabled` property is changed, not the page is loaded.
So I remove this because there are no possibility to take a long time to load the page.